common/page.go

```diff
@@ -1039,8 +1116,74 @@ func (p *Page) Workers() []api.Worker {
 	return workers
 }
 
+func (p *Page) onConsoleAPICalled(event *cdpruntime.EventConsoleAPICalled) {
+	// If there are no handlers for EventConsoleAPICalled, return
+	p.eventHandlersMu.RLock()
+	if _, ok := p.eventHandlers[eventPageConsoleAPICalled]; !ok {
+		p.eventHandlersMu.RUnlock()
+		return
+	}
+	p.eventHandlersMu.RUnlock()
+
+	m, err := p.consoleMssgFromConsoleEvent(event)
```

Comment:
```suggestion
	m, err := p.consoleMsgFromConsoleEvent(event)
```

Reply:
Force pushed with the change in https://github.com/grafana/xk6-browser/pull/1006/commits/12f6f4a6038e20c250fac2498f3cc37ecdf25b6a. See https://github.com/grafana/xk6-browser/blob/f5abd5d865238b52bc046a18ed082dcdfe4ed138/common/page.go#L1148.